tests/whist/core/cards/test_hand.py

```diff
@@ -6,24 +6,24 @@
 
 class HandTestCase(unittest.TestCase):
     def setUp(self) -> None:
-        self.hand = Hand()
+        self.hand = Hand.empty()
         self.spades_king = Card(suit=Suit.SPADES, rank=Rank.K)
 
     def contains_suit(self):
-        hand = Hand()
+        hand = Hand.empty()
         hand.add(Card(suit=Suit.HEARTS, rank=Rank.NUM_8))
         self.assertTrue(hand.contains_suit(Suit.HEARTS))
 
     def contains_not_suit(self):
-        hand = Hand()
+        hand = Hand.empty()
         hand.add(Card(suit=Suit.CLUBS, rank=Rank.NUM_8))
         self.assertTrue(hand.contains_suit(Suit.HEARTS))
 
     def test_empty(self):
-        self.assertEqual(Hand(), Hand.empty())
+        self.assertEqual(Hand(cards=()), Hand.empty())
```

Comment:
Do we have to check if the length is zero?

Reply:
i dont know? this is your test :P